### PR TITLE
Added Nitro Repair art/functionality

### LINKED ISSUE #186: Game: Implement Nitro Repair (medium)
**Describe the solution you'd like**
- Custom Particles so you can see when it is used
- Restore up to some number of hit points
- Gain a 30% speed boost for 3 seconds

### PR DESCRIPTION
**Pull request by SecondThread**
**Description**
Added nitro repair art and functionality. There is a casus variable you can set, and after that you will start using nitro repair. Here's what it looks like:
NitroRecover

This currently doesn't restore health because the system for that hasn't been made yet

Resolves #186 

**Type of change**
Check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Fix or feature that would cause existing functionality to not work as expected
- [ ] This change requires a update in the design document

**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.
passes flow.
works as shown in the GIF

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the design document (if applicable)
- [x] My changes generate no new warnings
- [x] Pulled updated master branch (if changed since branch creation) and corrected any conflicts, if any occur.

### CODE REVIEW

File: src/battleground/Battleground.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Seg from '../geometry/Seg.js';
 import GameObject from './GameObject.js';
 import {getTestTank} from '../tanks/TankLoader.js';
 
-const FPS=10;
+const FPS=30;
```
Comment (Derplime) [inline src/battleground/Battleground.js]: wooo no more slow gameeee

**Review by Derplime — APPROVED** [review]
Looks good to me. Anyone else is free to look at this too otherwise just merge it.

**Review by blaiadam — APPROVED** [review]
Looks great. Going to have to update it in the future to actually repair.

**Review by blaiadam — CHANGES_REQUESTED** [review]
Small import error needs fixing to remove this warning.

image

**Review by blaiadam — APPROVED** [review]
Awesome, thanks for fixing.

**Pull request merged**